src/components/UserInterface/UserDisplay/UserDisplay.js

```diff
@@ -3,12 +3,33 @@ import UserAccount from "../../SearchResults/UserAccount/UserAccount";
 import UserContact from "../../SearchResults/UserContact/UserContact";
 import styles from "./UserDisplay.module.css";
 
-const UserDisplay = ({ icons, userData: { profile, account, contact } }) => {
+const UserDisplay = ({ userData }) => {
+  const {
+    avatar_url,
+    name,
+    login,
+    created_at,
+    bio,
+    public_repos,
+    followers,
+    following,
+    location,
+    twitter_username,
+    blog,
+    company,
+  } = userData;
+
+  const displayData = {
+    profile: [avatar_url, name, login, created_at, bio],
```

Comment:
Have them in an object. They are not getting destructured correctly because you have the data in an array
```suggestion
    profile: {avatar_url, name, login, created_at, bio},
```

Reply:
I am not very clear on this @gertie-sheshe .. I understand I have destructured wrongly.. However, can I have an object without values as you have suggested? as objects should contain key-value pairs. Something coming to mind is doing an array destructuring in the child components instead. What do you think of this?

Reply:
Don't answer this. I have seen that array destructuring in the child component wouldn't work and having the data in an object instead does work. 
However in my experiments, I encountered some new errors that I found very confusing. 
I have taken screenshots of what I did and the unexpected results I got. I have also booked a session for us to go over why what worked and what didn't. Thanks